.claude/skills/test-coverage-for-feature/SKILL.md

```
@@ -0,0 +1,109 @@
+uv sync && uv run pytest -v
+cd ..
+actionlint .github/workflows/ci.yml  # if installed
+```
+
+Any failure: fix the root cause, do not skip or `--no-verify` anything. Re-run from the failed step.
+
+## Step 6 — commit and open the PR
+
+Atomic conventional commits. The pattern from the harness PR:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)